### ISSUE TITLE
Windows builds are broken in v1.2.3

### ISSUE THREAD

**Issue by buggins** [BindBC/bindbc-freetype#16]
It looks like windows builds are broken.
In dlangui CI targets latest-dmd and latest-ldc are failing.

```Building bindbc-common 0.1.3: building configuration [noBC]
    Building bindbc-loader 1.1.5: building configuration [noBC]
    Building bindbc-freetype 1.2.3: building configuration [dynamic]
C:\Users\runneradmin\AppData\Local\dub\packages\bindbc-freetype\1.2.3\bindbc-freetype\source\bindbc\freetype\dynload.d-mixin-74(74,9): Error: undefined identifier `bindModuleSymbols` in module `ft.bzip2`
Error C:\hostedtoolcache\windows\dc\ldc2-1.37.0\x64\ldc2-1.37.0-windows-multilib\bin\ldc2.exe failed with exit code 1.
```

Linux and OSX builds are ok.

**Comment by ichordev** [comment]
Ah, another instance of me putting a conditional compile statement in the wrong place! 🤦‍♀️ I’ll fix it when I’m home.

**Comment by ichordev** [comment]
Sorry for the delay! It should be fixed in v1.2.4.

**Comment by buggins** [comment]
Looks like it's fixed now. After changing of bindbc-freetype version in dlangui to v1.2.4 windows CI builds were fixed.
Thank you!
